frog-protocols/frog-fifo-v1.xml

```diff
@@ -92,8 +92,8 @@
         it sets a "fifo_barrier" condition on the surface associated with
         the fifo object. The condition is cleared immediately after the
         following latching deadline for non-tearing presentation.
-        If the surface is not currently being presented by the compositor,
-        the condition needs to be cleared soon enough so that forward
+        If no latching deadline is scheduled after the content update has been
+        applied, the condition needs to be cleared soon enough so that forward
         progress for the application is guaranteed. The exact rate at which
```

Comment:
This could still be misinterpreted if you pick it apart like this, a deadline being scheduled doesn't say anything about it happening soon enough. Maybe we should just remove that part of the sentence, like

```suggestion
        The condition needs to be cleared soon enough so that forward
        progress for the application is guaranteed. The exact rate at which
```

Reply:
Done